src/foam/u2/borders/SplitScreenGridBorder.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'SplitScreenGridBorder',
+  extends: 'foam.u2.Element',
+
+  imports: [
+    'displayWidth'
+  ],
+
+  requires: [
+    'foam.u2.layout.Grid',
+    'foam.u2.layout.GUnit'
+  ],
+
+  css: `
+    ^{
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      padding: 0 4vw;
+      background-color: /*%WHITE%*/ white;
+
+      /* minus footer */
+      height: calc(100% - 65px);
+      height: -moz-calc(100% - 65px);
+      height: -webkit-calc(100% - 65px);
+    }
+
+    ^grid {
+      grid-gap: clamp(1vmax, 1.5vmax, 2vmax) clamp(1vmax, 1.5vmax, 2vmax);
```

Comment:
If gaps in both directions are the same then we only need to provide one value. Also the min and max value for clamps should be absolute cause relative values means that it will always pick the middle. Maybe `2rem` and `10rem` for this case

Reply:
Addressed, thanks. i used 1 and 5 instead since with 10 the grid gap was taking over the entire page